subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -90,6 +90,9 @@ dependencies {
     androidTestImplementation(project(":subprojects:common:okhttp"))
     androidTestImplementation(project(":subprojects:common:time"))
 
+    androidTestImplementation(Dependencies.androidTest.runner)
```

Comment:
How did it happen?

Reply:
Happened what? Tests work only because the orchestrator was already installed on the emulator. For instance, when running Avito test.

Reply:
Understand. Do we need Implementation dependency or can replace it with Api?

Reply:
`implementation` dependency exposes less, thus it's preferable.
Also, it is in the application. No one can consume this module as dependency

Reply:
And according to [doc](https://developer.android.com/training/testing/junit-runner#ato-gradle) we should use `implementation` not `api`.

Reply:
Oh, wait. Implementation exposes more.

Reply:
https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation
```The plugin exposes two configurations that can be used to declare dependencies: api and implementation. The api configuration should be used to declare dependencies which are exported by the library API, whereas the implementation configuration should be used to declare dependencies which are internal to the component```
I think we need to use it like api

Reply:
Why? It's a test application. Why should it export its dependencies like a library? Orchestrator is an internal dependency, so according to a piece of documentation you provided we should use `implementation`.

Reply:
It is controversial. I was led by the principle that fewer dependencies are preferable. If we need something let add it explicitly

Reply:
So, we discussed it in internal chat and realize that `implementation` and `api` influence only to consumers of a library.